timed/employment/serializers.py

```diff
@@ -91,7 +91,21 @@ def get_worktime_balance_raw(self, instance):
             timedelta()
         )
 
-        return reported_worktime + overtime_credit - expected_worktime
+        absences = sum(
+            Absence.objects.filter(
+                user=instance,
+                date__gte=start_date,
+                date__lte=end_date
+            ).values_list('duration', flat=True),
+            timedelta()
+        )
+
+        return (
+            reported_worktime +
+            absences +
```

Comment:
I think the absences should be documented in the docstring of this method as well.